index.ts

```diff
@@ -1,105 +1,40 @@
 import { version } from './package.json';
+import { EventTypes, Options, WidgetEvents, SetThemeParameters } from './types';
 
 const externalStaticOrigin = 'https://javascript.wert.io';
 
-interface options {
-  partner_id?: string
-  origin?: string
-  address?: string
-  theme?: string
-  currency?: string
-  currency_amount?: number
-  commodity?: string
-  commodity_amount?: number
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  [x: string]: any
-}
-
-interface extraOptions {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  [x: string]: any
-}
-
-interface eventOptions {
-  type: string
-  origin: string
-  data: {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    [x: string]: any
-  }
-}
-
-type customListener = (data: {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  [x: string]: any
-}) => void;
-
-type setThemeData = {
-  theme?: string
-  colors?: {
-    [x: string]: string
-  }
-};
-
 class WertWidget {
-
   private iframe: HTMLIFrameElement = document.createElement('iframe');
-
-  partner_id?: string;
-  origin: string;
-  options: options;
-  extraOptions: extraOptions;
-  listeners: {
-    [x: string]: customListener;
-  };
-  widgetWindow: Window | null;
-  checkIntervalId: number | undefined;
-
-
-  static get eventTypes(): string[] {
```

Comment:
Did you decide to remove this method so that the readme remains the single source of truth?